### PR TITLE
add a sleep to the puma restart using a random integer between 0 and …

### PR DESCRIPTION
**Pull request by mshepet**
…180 seconds

should help with high request queuing during deploys

### CODE REVIEW

**Comment by SudoNikon** [comment]
👍

File: opsworks-puma/recipes/deploy.rb [hunk 1]
```diff
@@ -16,6 +16,9 @@
 
   service application do
     if node[:opsworks][:activity] == "deploy"
+      random = Random.new.rand(0..180)
+      Chef::Log.info("Sleeping #{random} seconds")
```
Comment (sb8244) [inline opsworks-puma/recipes/deploy.rb]: Is this rolling restarts?? 😆

Comment (mshepet) [reply opsworks-puma/recipes/deploy.rb]: Not the most elegant solution but will help stagger restarts for sure.

Comment (sb8244) [reply opsworks-puma/recipes/deploy.rb]: It's simple & smart. I like it.

**Comment by smitchelus** [comment]
Wow. Awesome Mike!

**Pull request merged**